Mlem/App/Views/Root/Tabs/Settings/AccountAdvancedSettingsView.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  AccountAdvancedSettingsView.swift
+//  Mlem
+//
+//  Created by Sjmarf on 13/10/2024.
+//
+
+import SwiftUI
+
+struct AccountAdvancedSettingsView: View {
+    @Environment(Palette.self) var palette
+    
+    @State var isBot: Bool = false
+    
+    init() {
+        guard let session = AppState.main.firstSession as? UserSession else {
```

Comment:
We might want to roll this functionality into the new `accountType` introduced in #1392. Fine to leave OOS.